scripts/install-calico-windows.ps1

```diff
@@ -15,30 +12,16 @@ layout: null
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{%- if site.prodname == "Calico" %}
-{%- assign installName = "Calico for Windows" %}
-{%- assign rootDir = "CalicoWindows" %}
-{%- assign zipFileName = "calico-windows.zip" %}
-{%- else %}
-{%- assign installName = "Tigera Calico for Windows" %}
-{%- assign rootDir = "TigeraCalico" %}
-{%- assign zipFileName = "tigera-calico-windows.zip" %}
-{%- endif %}
-
 <#
 .DESCRIPTION
-    This script installs and starts {{site.prodname}} services on a Windows node.
+    This script installs and starts Calico services on a Windows node.
 
     Note: EKS requires downloading kubectl.exe to c:\k before running this script: https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 #>
 
 Param(
-{%- if site.url == "https://docs.projectcalico.org" %}
-    [parameter(Mandatory = $false)] $ReleaseBaseURL="https://github.com/projectcalico/calico/releases/download/{{site.data.versions.first.components["calico/node"].version}}/",
-{%- else %}
-    [parameter(Mandatory = $false)] $ReleaseBaseURL="{{site.url}}/files/windows/",
-{%- endif %}
-    [parameter(Mandatory = $false)] $ReleaseFile="calico-windows-{{site.data.versions.first.components["calico/node"].version}}.zip",
+    [parameter(Mandatory = $false)] $ReleaseBaseURL="https://github.com/projectcalico/calico/releases/download/a0e0bcc9b518/",
+    [parameter(Mandatory = $false)] $ReleaseFile="calico-windows-a0e0bcc9b518.zip",
```

Comment:
I think the portion after `calico-windows` refers to the git hash right? Does that mean we'll need to keep this updated?

Reply:
The checked-in master version of this file will have this git hash in the $ReleaseFile. For this file on release branches, the $ReleaseFile will have a version as a suffix, e.g. `calico-windows-v3.21.2.zip`.

I don't think we need to keep this hash up to date on master. On master, this version of the script isn't (or shouldn't be) used.

Reply:
gotcha. Might be worth it to either add a comment or just put a dummy value there to make it more clear for people who've misunderstood like me.

Reply:
Good idea, I changed the version to `master` and added a comment that it doesn't actually exist (this has always been the case but now it's highlighted a bit more): 8a09b556f9e9c3d4fef3b581a79643866618a93d